core/commands/ipns.go

```diff
@@ -0,0 +1,104 @@
+package commands
+
+import (
+	"errors"
+	"io"
+	"strings"
+
+	cmds "github.com/ipfs/go-ipfs/commands"
+	namesys "github.com/ipfs/go-ipfs/namesys"
+	u "github.com/ipfs/go-ipfs/util"
+)
+
+var ipnsCmd = &cmds.Command{
```

Comment:
very minor thing-- i think we'll later want this to be something like `ipnsResolve` (later, doesnt matter), because i think `ipfs name` and `ipns` can be the same thing. (i.e. `ipnsCmd` would have both `publish` and `resolve` as children). thus an `ipns` independent tool could publish a record.